examples/ht16k33_segments_7x4customchars.py

```diff
@@ -0,0 +1,44 @@
+# SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
+# SPDX-License-Identifier: MIT
+
+# Basic example of setting digits on a LED segment display.
+# This example and library is meant to work with Adafruit CircuitPython API.
+# Author: Alec Delaney
+# License: Public Domain
+
+# Import all board pins.
+import time
+import board
+import busio
+from adafruit_ht16k33 import segments
+
+# Create the character dictionary
+# You can use the list normally referenced as a starting point
+custom_chars = {}
+typical_list_values = segments.NUMBERS
+typical_list_chars = list("0123456789abcdef ")
```

Comment:
NUMBERS do not contain space but "-". So in the example spaces are displayed as "-".
```suggestion
typical_list_chars = list("0123456789abcdef-")
```